quantulum3/_lang/en_US/tests/quantities.json

```diff
@@ -1384,5 +1384,166 @@
           "surface": "three million, two hundred & forty"
         }
       ]
+  },
+  {
+    "req": "I have seen 45 and 50 cases.",
+    "res": [
+        {
+          "value": 45,
+          "unit": "dimensionless",
+          "surface": "45"
+        },
+        {
+          "value": 50,
+          "unit": "dimensionless",
+          "surface": "50"
+        }
+    ]
+  },
+  {
+    "req": "They were given 45 and 50 mg ethanol.",
+    "res": [
+        {
+          "value": 45,
+          "unit": "milligram",
+          "surface": "45",
+          "entity": "mass",
+          "uncertainty": null
+        },
+        {
+          "value": 50,
+          "unit": "milligram",
+          "surface": "50 mg",
+          "entity": "mass",
+          "uncertainty": null
+        }
+    ]
+  },
+  {
+    "req": "They were given between 44 and 50 mg ethanol.",
+    "res": [
+        {
+          "value": 47,
+          "unit": "milligram",
+          "surface": "between 44 and 50 mg",
+          "entity": "mass",
+          "uncertainty": 3
+        }
+    ]
+  },
+  {
+    "req": "They were given between 44 mg and 50 mg ethanol.",
+    "res": [
+        {
+          "value": 47,
+          "unit": "milligram",
+          "surface": "between 44 mg and 50 mg",
+          "entity": "mass",
+          "uncertainty": 3
+        }
+    ]
+  },
+  {
+    "req": "They were given between 44 mg to 50 mg ethanol.",
+    "res": [
+        {
+          "value": 47,
+          "unit": "milligram",
+          "surface": "44 mg to 50 mg",
+          "entity": "mass",
+          "uncertainty": 3
+        }
+    ]
+  },
+  {
+    "req": "They were given 44 mg to 50 mg ethanol.",
+    "res": [
+        {
+          "value": 47,
+          "unit": "milligram",
+          "surface": "44 mg to 50 mg",
+          "entity": "mass",
+          "uncertainty": 3
+        }
+    ]
+  },
+  {
+    "req": "They were given 44 mg to 50 mg ethanol and ten pills.",
+    "res": [
+        {
+          "value": 47,
+          "unit": "milligram",
+          "surface": "44 mg to 50 mg",
+          "entity": "mass",
+          "uncertainty": 3
+        },
+        {
+          "value": 10,
+          "unit": "dimensionless",
+          "surface": "ten"
+        }
+    ]
+  },
+  {
+    "req": "They were given from 44 mg to 50 mg ethanol.",

```

Comment:
This example seems wrong to me intuitively and is not consistent with the code docs

Reply:
The cases I had in mind that preclude merging `from X to Y` are like the one below:

`Post-crop steam treatment ( 60 degrees C , 24 h ) of SMS reduced phytase activity from 0.074 to 0.018 unit/g .`

if it is less confusing, we can change the test string.

or we can disable this "from" check if you prefer.

Reply:
I think the test string is great, because there is an underlying issue here that should be solved!
My counter example that this should be a range (another one, next to https://github.com/nielstron/quantulum3/pull/204#discussion_r1063241654) : "This means that the USDA hardiness zones relevant to Britain and Ireland are quite high, from 7 to 10, as shown below." (https://en.wikipedia.org/wiki/Hardiness_zone#Britain_and_Ireland) which seems like it should definitely be a range.

Reply:
Hence there are cases where it might mean a range with uncertainty and sometimes a range where we are actually interested in upper/lower bound.

Reply:
Yup. I am not sure how to disambiguate these cases properly though.. I will
defer to your judgement as to how to handle this (can help with coding a
solution if needed).